packages/gamut-labs/src/DropdownButton/DropdownList.tsx

```diff
@@ -24,19 +25,25 @@ const StyledAnchor = styled(Anchor)`
 
 export const DropdownList: React.FC<DropdownListProps> = ({
   dropdownItems,
+  onClose,
 }) => {
   return (
     <FlexBox flexDirection="column" borderRadius="2px" bg="white" py={12}>
       {dropdownItems.map((item) => {
         const Icon = item.icon;
         const { id, text, href, clickHandler } = item;
 
+        const onClick = (event: React.MouseEvent) => {
+          clickHandler && clickHandler(event);
+          onClose && onClose();
```

Comment:
One of these days I'll get that lint rule up and running 😄 

```suggestion
          clickHandler?.(event);
          onClose?.();
```